generators/entity-client/templates/react/src/main/webapp/app/entities/entity.tsx.ejs

```diff
@@ -107,15 +107,7 @@ export const <%= entityReactName %> = (props: I<%= entityReactName %>Props) => {
     <%_ } else { _%>
     props.getEntities();
     <%_ } _%>
-  }, [<%_ if (searchEngine === 'elasticsearch' && pagination === 'no') { _%>search<%_ } _%>]);
-  <%_ } _%>
-
-  <%_ if (pagination === 'infinite-scroll') { _%>
-  useEffect(() => {
-    if (props.updateSuccess) {
-      getAllEntities();
-    }
-  });
+  }, []);
```

Comment:
@qmonmert : While you are at this I think you can remove the `<%_ if (pagination === 'infinite-scroll') { _%>` statement as I think it's unnecessary. After all if `<%_ if (pagination !== 'no') { _%>` it implies that `<%_ if (pagination === 'infinite-scroll') { _%>` :smile:

Reply:
indeed I think you are right, I do that 👍